apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -445,6 +488,7 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
         childrenClassName="lg:ml-0"
         title={t("offset_toggle")}
         description={t("offset_toggle_description")}
+        {...offsetStartLockedProps}
```

Comment:
This was removed recently to allow these fields to be unlocked for v1, now with v2 we need them as the team admin can discretely select their state